src/credentialshandler.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2016  Aditya Dev Sharma <aditya.sharma15696@gmail.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ */
+
+#include "credentialshandler.h"
+
+#include <QtCore/QCoreApplication>
+#include <QtCore/QMap>
+
+CredentialsHandler::CredentialsHandler(QObject* parent): QObject(parent)
+{
+
+}
+
+void CredentialsHandler::showUserCredentialsPage()
+{
+    m_view.setSource(QUrl::fromLocalFile("UserCredentials.qml"));
```

Comment:
Here, the QUrl::fromLocalFile does not work with this details although. When I enter my "system-specific" url it works as expected. (and that is how I tested everything). This needs to be found a way around